Front-End/Javascript/Javascript-Basics/doWhileLoop.js

```diff
@@ -0,0 +1,5 @@
+var i = 0;
+
+do{
+    console.log("Would run atleast once");
```

Comment:
This would run at least once even if the condition is false at the very start itself